otamodel/OTA_HotelProductNotifRQ.go

```diff
@@ -71,57 +131,65 @@ type StayDateRange struct {
 	End      string `json:"End,omitempty"`
 }
 
-//HotelProductRatePlan **
-type HotelProductRatePlan struct {
-	RatePlanCode  string          `json:"RatePlanCode,omitempty,required"`
-	RatePlanName  string          `json:"RatePlanName,omitempty,required"`
-	RatePlanType  string          `json:"RatePlanType,omitempty"`
-	CurrencyCode  string          `json:"CurrencyCode,omitempty,required"`
-	BaseOccupancy string          `json:"BaseOccupancy,omitempty,required"`
-	Descriptions  Descriptions    `json:"Descriptions,omitempty,required"`
-	StayDateRange []StayDateRange `json:"StayDateRange,omitempty,required"`
-}
-
-//HotelProductRatePlans **
-type HotelProductRatePlans struct {
-	RatePlan *[]HotelProductRatePlan `json:"RatePlan,omitempty,required"`
-}
-
 //PrepaymentPolicy **
 type PrepaymentPolicy struct {
 	EffectiveFrom string `json:"EffectiveFrom,omitempty"`
 }
 
 //TPAExtensions **
 type TPAExtensions struct {
-	PrepaymentPolicy PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
+	PrepaymentPolicy *PrepaymentPolicy `json:"PrepaymentPolicy,omitempty"`
 }
 
 //GuaranteePayment **
 type GuaranteePayment struct {
-	TPAExtensions    *TPAExtensions    `json:"TPA_Extensions,omitempty"`
-	PolicyCode       string            `json:"PolicyCode,omitempty"`
-	AcceptedPayments *AcceptedPayments `json:"AcceptedPayments,omitempty"`
+	TPAExtensions    *TPAExtensions        `json:"TPA_Extensions,omitempty"`
+	AmountPercent    *PenaltyAmountPercent `json:"AmountPercent,omitempty"`
+	PolicyCode       string                `json:"PolicyCode,omitempty"`
+	GuaranteeType    string                `json:"GuaranteeType,omitempty"`
+	HoldTime         string                `json:"HoldTime,omitempty"`
+	Start            string                `json:"Start,omitempty"`
+	End              string                `json:"End,omitempty"`
+	Mon              string                `json:"Mon,omitempty"`
+	Tue              string                `json:"Tue,omitempty"`
+	Weds             string                `json:"Weds,omitempty"`
+	Thur             string                `json:"Thur,omitempty"`
+	Fri              string                `json:"Fri,omitempty"`
+	Sat              string                `json:"Sat,omitempty"`
+	Sun              string                `json:"Sun,omitempty"`
+	PaymentCode      string                `json:"PaymentCode,omitempty"`
+	AcceptedPayments *AcceptedPayments     `json:"AcceptedPayments,omitempty"`
```

Comment:
OTA_HotelProductNotifRQ message received from CM not supporting AcceptedPayments struct. 
Please remove it.

Reply:
Done